soepy/test/test_regression.py

```diff
@@ -16,14 +17,15 @@ def test1(idx):
     with open(vault, "rb") as file:
         tests = pickle.load(file)
 
-    test = tests[idx]
+    random.seed(787)
```

Comment:
Why do we need to set a seed here again? This is taken care of [here](https://github.com/OpenSourceEconomics/soepy/blob/5bba1f24f7ccfa4bc57c6f359c28588b54da4f50/soepy/test/conftest.py#L10) already, right?